### PR TITLE
Remove the browser VC that is no longer used

### PR DESCRIPTION
**Pull request by coreyfloyd**
And the associated tweak

### CODE REVIEW

**Comment by joewalsh** [comment]
👍

File: Wikipedia/Code/UIViewController+WMFArticlePresentation.h [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+@class MWKTitle, MWKDataStore;
+@class WMFTableOfContentsViewController;
+
+NS_ASSUME_NONNULL_BEGIN
+
+
+@interface UIViewController (WMFArticlePresentation)
+
+- (void)wmf_pushArticleWithTitle:(MWKTitle*)title dataStore:(MWKDataStore*)dataStore restoreScrollPosition:(BOOL)restoreScrollPosition animated:(BOOL)animated;
```
Comment (montehurd) [inline Wikipedia/Code/UIViewController+WMFArticlePresentation.h]: I don't see anywhere these new methods are being called in this diff...

Comment (coreyfloyd) [inline Wikipedia/Code/UIViewController+WMFArticlePresentation.h]: @montehurd these are not new methods - this category was just moved from the browser view class that was deleted. Check out the deleted part of the diff to see.

(No behavior was changed in this diff… just the browser code path and tweak were removed)

Comment (montehurd) [inline Wikipedia/Code/UIViewController+WMFArticlePresentation.h]: Ah, gotcha.

**Comment by montehurd** [comment]
👍

**Comment by coreyfloyd** [comment]
Dont merge!

**Pull request merged**